pkg/scheduling/requirements.go

```diff
@@ -220,15 +220,25 @@ func editDistance(s, t string) int {
 }
 
 func labelHint(r Requirements, key string, allowedUndefined sets.Set[string]) string {
+	splitKey := strings.Split(key, "/")
+	suffix := splitKey[1]
```

Comment:
This crashes if there isn't a `/` in the key. We should _either_ split the string if a `/` exists or take the entire string if it doesn't. That way, if they forget to provide a prefix, we can also suggest something to them.